tests/framework/unit_tests/CustomDrivers/basic_pip.xml

```diff
@@ -0,0 +1,93 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/CodeInterfaceTests/RAVEN.Basic</name>
+    <author>dgarrett622</author>
+    <created>2022-04-20</created>
+    <classesTested>Models.Code.RAVEN</classesTested>
+    <description>
+      Basic test of the raven-runs-raven interface for running in Python workflows (same as integrated
+      RAVEN runs RAVEN basic test). Outer raven runs a grid sampling by changing the upper and lower
+      bounds of the sampledVars distribution on the inner raven. Inner raven runs 100 MC samples,
+      does basic stats, and returns the mean, sigma to the outer raven.
+    </description>
+    <analytic>
+      Each sample in this test converges towards the analytic solutions discussed in the "changing lower, upper bounds" section of
+      the "attenuate" analytic model documentation.
+    </analytic>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>Basic</WorkingDir>
+    <Sequence>sample,print</Sequence>
+  </RunInfo>
+
+  <Steps>
+    <MultiRun name="sample">
+      <Input class="Files" type="raven">inner_input</Input>
+      <Model class="Models" type="Code">raven</Model>
+      <Sampler class="Samplers" type="Grid">grid</Sampler>
+      <Output class="DataObjects" type="PointSet">outer_samples</Output>
+    </MultiRun>
+    <IOStep name="print">
+      <Input class="DataObjects" type="PointSet">outer_samples</Input>
+      <Output class="OutStreams" type="Print">outer_out</Output>
+    </IOStep>
+  </Steps>
+
+  <Files>
+    <Input name="inner_input" type="raven">../../../CodeInterfaceTests/RAVEN/Basic/inner_sample.xml</Input>
+  </Files>
+
+  <Models>
+    <Code name="raven" subType="RAVEN">
+        <executable>raven_framework</executable>
```

Comment:
Could %RAVENFRAMEWORK% be used here, and then we could just use one test instead of duplicating tests?

Reply:
I also don't like the duplication I've had to do with this test. The hurdle here is knowing how to run the inner RAVEN if the outer RAVEN was run with the PythonRaven interface. There's no clear way to recycle the outer run strategy for the inner in that case. That leaves us with manually specifying the way to run RAVEN for the inner, which is different for the different install types, hence the split test. All that said, if you have any ideas on how we can set this up without the duplication, it would certainly be preferable. I just can't think of how to reasonably set that up.